pkg/controller/controller.go

```diff
@@ -675,6 +675,7 @@ func (h *Handler) generateLocalProvisionerDaemonset(cr *localv1.LocalVolume) *ap
 				Spec: corev1.PodSpec{
 					Containers:         containers,
 					ServiceAccountName: provisionerServiceAccount,
+					Tolerations:	 	cr.Spec.Tolerations,
```

Comment:
Whitespaces are odd here (or bad gihub rendering?)

Reply:
Filed #60 to get go fmt in PR tests

Reply:
These should now be corrected.